features/features/package_managers/mix_spec.rb

```diff
@@ -12,16 +12,18 @@
     LicenseFinder::TestingDSL::MixProject.create
     puts 'mix project created'
     elixir_developer.run_license_finder
-    expect(elixir_developer).to be_seeing_line 'fs, 0.9.1, ISC'
+    expect(elixir_developer).to be_seeing_line 'fs, 0.9.1, MIT'
```

Comment:
The [hex package](https://github.com/synrc/fs/blob/master/package.exs#L17) says the license is MIT